SingularityService/src/main/java/com/hubspot/singularity/scheduler/SingularityUsagePoller.java

```diff
@@ -26,23 +33,33 @@
   private final UsageManager usageManager;
   private final SingularityUsageHelper usageHelper;
   private final SingularityExceptionNotifier exceptionNotifier;
+  private final RequestManager requestManager;
 
   @Inject
-  SingularityUsagePoller(SingularityConfiguration configuration, SingularityUsageHelper usageHelper, UsageManager usageManager, MesosClient mesosClient, SingularityExceptionNotifier exceptionNotifier) {
+  SingularityUsagePoller(SingularityConfiguration configuration,
+                         SingularityUsageHelper usageHelper,
+                         UsageManager usageManager,
+                         MesosClient mesosClient,
+                         SingularityExceptionNotifier exceptionNotifier,
+                         RequestManager requestManager) {
     super(configuration.getCheckUsageEveryMillis(), TimeUnit.MILLISECONDS);
 
     this.configuration = configuration;
     this.usageHelper = usageHelper;
     this.mesosClient = mesosClient;
     this.usageManager = usageManager;
     this.exceptionNotifier = exceptionNotifier;
+    this.requestManager = requestManager;
   }
 
   @Override
   public void runActionOnPoll() {
     final long now = System.currentTimeMillis();
+    Map<RequestType, Map<String, Number>> usagesPerRequestType = new HashMap<>();
```

Comment:
This should be per slave. This poller loops through each slave and creates a new `SingularitySlaveUsage` with the stats for that slave